common/src/main/java/com/distkv/dst/common/exception/SortedListDuplicatedMembersException.java

```diff
@@ -0,0 +1,27 @@
+package com.distkv.dst.common.exception;
+
+public class SortedListDuplicatedMembersException extends DstException {
+
+  protected String errorCode = "008";
```

Comment:
This should be represented by a separate enumeration class to avoid magic numbers

Reply:
+1

Reply:
I get.